03-Bash/fix_owner/fix_owner.sh

```diff
@@ -54,13 +55,46 @@ get_file_grup(){
 	echo $(ls -nd "$1" | cut --delimiter=" " -f 4)
 }
 
+#1-dir 2-file_check_list
+creat_list_for_check(){
+	echo $1 > $2
+	find $1 -name "*" | sed 's/ /\\ /g' >> $2
+}
+
+#1-dir/file 2-USER 3-ID 4-GRUP
+check_item(){
+	if [ -d $1 ]; then
+		local FILE_ID_USER=$(get_file_id $1)
+		local FILE_GRUP_USER=$(get_file_grup $1)
+
+		if [ $FILE_ID_USER -ne $3 ] || [ $FILE_GRUP_USER -ne $4 ]; then
+			if [ "$_force" = true ]; then
+				echo chown $3:$4 $1
+			else
+				if [ $FILE_ID_USER -ne $3 ]; then
+					echo "The directory $1 doesn't belong to the user $2 with id $3 !"
+				fi
+				if [ $FILE_GRUP_USER -ne $4 ]; then
+					echo "The directory $1 doesn't belong to the user $2 with grup $4 !"
+				fi
+
+				read -p "To fix? [y/n]" _key < /dev/tty
```

Comment:
This implementation tried to change every file instead of changing id and group for selected files.